docs/semgrep-code/triage-remediation.md

```diff
@@ -149,112 +149,26 @@ To **reopen multiple findings** in the **No grouping** view, follow these steps:
 
 Triage your Semgrep AppSec Platform findings displayed as comments in PRs and MRs by replying with another comment.
 
-<Tabs
-    defaultValue="gh"
-    values={[
-        {label: 'Azure DevOps', value: 'ado'},
-        {label: 'Bitbucket', value: 'bb'},
-        {label: 'GitHub', value: 'gh'},
-        {label: 'GitLab', value: 'gl'}
-    ]}
->
+Before proceeding, ensure that you have:
+    - One or more repositories hosted by a source code manager supported by Semgrep. See [PR or MR comments](/category/pr-or-mr-comments) for requirements information specific to your source code manager (SCM).
+    - Completed a [Semgrep core deployment](/deployment/core-deployment)
```

Comment:
What about this:

```
Before proceeding, ensure that you have:
  - One or more repositories hosted by a source code manager supported by Semgrep
  - Configured [PR or MR comments](/category/pr-or-mr-comments) for your source code manager (SCM)
 ```
Helps with punctuation consistency and focuses on the critical part of the core setup, I think?